packages/api/cms-api/src/auth/strategies/static-authed-user.strategy.ts

```diff
@@ -19,9 +19,9 @@ export function createStaticAuthedUserStrategy(config: StaticAuthedUserStrategyC
                 secretOrKey,
             });
         }
-
-        validate(data: CurrentUserInterface): CurrentUserInterface {
-            return data;
+        validate(): CurrentUserInterface | undefined {
+            if (typeof config.staticAuthedUser !== "object") return undefined;
```

Comment:
Should we throw an error (e.g. "Missing validate implementation") instead?

Reply:
Not necessary, this is done in the guard: https://github.com/vivid-planet/comet/blob/main/packages/api/cms-api/src/auth/guards/global.guard.ts#L31